query/src/interpreters/interpreter_copy.rs

```diff
@@ -57,47 +52,47 @@ impl Interpreter for CopyInterpreter {
         &self,
         mut _input_stream: Option<SendableDataBlockStream>,
     ) -> Result<SendableDataBlockStream> {
-        let table = self
-            .ctx
-            .get_table(&self.plan.db_name, &self.plan.tbl_name)
-            .await?;
-
-        let location = self.plan.location.clone();
-        let c = extract_stage_location(location.as_str());
-        if c.is_err() {
-            return Err(ErrorCode::BadOption(
-                "Cannot convert value to stage and path",
-            ));
-        }
-        let (stage, path) = c.unwrap();
+        tracing::info!("Plan:{:?}", self.plan);
+
+        let plan = self.plan.stage_plan.clone();
+        let source_stream = match plan.stage_info.stage_type {
+            StageType::External => {
+                match plan.stage_info.file_format_options.format {
+                    // CSV.
+                    StageFileFormatType::Csv => {
```

Comment:
It seems we support `parquet` too?

Reply:
Yes, I will add the parquet support after the csv works all well  :D